### ISSUE TITLE
About training loss in paper and implementation

### ISSUE THREAD

**Issue by SuwoongHeo** [garvita-tiwari/neuralgif#10]
Hi, 

In the paper, you said that the pretaining is conducted with the supervision of blend weights only (`wgt` in the below code). However, there are several losses except the skinning weight loss named `diff_can` and `spr_wgt`. https://github.com/garvita-tiwari/neuralgif/blob/40a4a96c234aaaa795770e25ae10be958b56932f/models/train_shape.py#L156-L160

The definition of `diff_can` is obvious but `spr_wgt` defined below is hard to understand for me. Could you give me an explanation about what this loss means or any reference related to this loss?
https://github.com/garvita-tiwari/neuralgif/blob/40a4a96c234aaaa795770e25ae10be958b56932f/models/train_shape.py#L152-L153

Thanks in advance :)
